gulpfile.js

```diff
@@ -32,27 +31,26 @@ gulp.task('run', () => {
         'node_modules/'
       ]
     });
-
-    setTimeout(() => {
-      const publicUrl = `${url.replace('https://', 'http://')}`;
-      util.log('Public Url:', publicUrl);
-
-      util.log('Patching rule on tenant.');
-      getCurrentConfig().then((config) => {
+    setTimeout(async () => {
+      try {
+        const publicUrl = `${url.replace('https://', 'http://')}`;
+        util.log('Public Url:', publicUrl);
+        util.log('Patching rule on tenant.');
         const adapter = new ManagementClientAdapter(config);
-        install(adapter, {
+        await install(adapter, {
           extensionURL: publicUrl,
           username: 'Development',
           clientID: config.AUTH0_CLIENT_ID,
           clientSecret: config.AUTH0_CLIENT_SECRET
-        })
-          .then(() => {
-            util.log('Rule patched on tenant.');
-          })
-          .catch((error) => {
-            util.log("Couldn't patch rule in tenant:", error);
-          });
-      });
+        });
+        util.log('Rule patched on tenant.');
+      } catch (error) {
+        util.log("Couldn't patch rule in tenant:", error);
+      }
     }, 4000);
-  });
-});
+  } catch (error) {
+    throw error;
```

Comment:
why catch if you're just re-throwing?

Reply:
No good reason. I can remove the catch because it's redundant.